appium-dotnet-driver/Appium/AppiumDriver.cs

```diff
@@ -431,6 +429,38 @@ public object GetSessionDetail(string detail)
 
         #endregion Session Data
 
+        #region Device Methods
+
+        /// <summary>
+        /// Start recording the device screen
+        /// </summary>
+        /// <param name="opts">recording options like the following:
+        ///  Dictionary<string, object> StartRecordOptions = new Dictionary<string, object>();
+        ///  StartRecordOptions.Add("bit_rate", "1");
+        ///  StartRecordOptions.Add("video_size", "1280x720");
+        /// </param>
+        public void StartRecordingScreen(Dictionary<string, object> opts)
+        {
+            Dictionary<string, object> parameters = new Dictionary<string, object>();

```

Comment:
why it is necessary to copy all values?

Reply:
I copied the Rotate command here, 

```
 public void Rotate(Dictionary<string, int> opts)
        {
            Dictionary<string, object> parameters = new Dictionary<string, object>();
            foreach (KeyValuePair<string, int> opt in opts)
            {
                parameters.Add(opt.Key, opt.Value);
            }
            Execute(AppiumDriverCommand.Rotate, parameters);
        }

```


would you prefer to have a statically typed object? like the Location command?